tools/dbgen.py

```diff
@@ -377,13 +388,34 @@ def generate_method_accessors(obj: Object) -> CodeLines:
             default_str = f', {'true' if default else 'false'}'
         else:
             default_str = f', {default}'
+
+        if default_str != '':
+            default_str_doxy=', default {default_str}'
+        else:
+            default_str_doxy=''
+
         lines.header += [
             '',
+            f'/**',
+            f' * @brief get{make_typename(key)}',
+            f' * ',
+            f' * get the value for {obj.path}.{key}',
+            f' * ',
+            rf' * @return `{ctype}`; the value for {obj.path}.{key}{default_str_doxy}',
+            f' */',
             f'{ctype} get{make_typename(key)}() const',
             '{',
             [f'return getValue<{ctype}>({get_string(key)}{default_str});'],
             '}',
             '',
+            f'/**',
+            f' * @brief set{make_typename(key)}',
+            f' * ',
+            f' * set the value for {obj.path}.{key}',
+            f' * ',
+            rf' * @param value the `{ctype}` value for {obj.path}.{key}{default_str}',
+            f' * @return bool success',
+            f' */',
```

Comment:
With annotation we'd have something like this:

```
 '/**',
f' * @brief {obj.title}',
 ' * ',
f' * {obj.description}',
 ' */',
```

If there's no annotation, then is there any point in adding a header? Could add a `this object requires annotating` or similar.